src/Core/Domain/Customer/ValueObject/ApeCode.php

```diff
@@ -60,7 +60,7 @@ public function getValue()
     private function assertIsApeCode($code)
     {
         if (!is_string($code)
-            || (!empty($code) && !((bool) preg_match('/^\d{3,4}[a-zA-Z]{1}$/', $code)))
+            || (!empty($code) && !((bool) preg_match('/^[0-9]{1,2}?\.?[0-9]{1,2}[a-zA-Z]{1}$/s', $code)))
```

Comment:
The same regex is used at many places, maybe you can make a constant ?

Reply:
Done 😄